Makefile

```diff
@@ -9,7 +9,7 @@ format:
 
 .PHONY: test
 test:
-	uv run pytest
+	DATABASE_URL=sqlite+aiosqlite:///database_test.sqlite uv run pytest
```

Comment:
Not sure about that strategy, there might be [a better one](https://stackoverflow.com/questions/36141024/how-to-pass-environment-variables-to-pytest) (either using an [autouse fixture](https://stackoverflow.com/a/74888005) or [pytest env](https://pypi.org/project/pytest-env/)). The main drawback of setting this in the Makefile is when we want to launch pytest manually. WDYT?